cpp/Json.hpp

```diff
@@ -0,0 +1,405 @@
+#pragma once
+
+/// @brief JSON stream parser
+
+#include <queue>
+#include <cstring>
```

Comment:
Checking if a character is a space is the only thing you use from there

Reply:
I'll try to integrate this into `CString` but I may need to lookup how Unicode does that